lib/foreman_maintain/utils/command_runner.rb

```diff
@@ -59,12 +59,15 @@ def execution_error
 
       private
 
+      # rubocop:disable Metrics/MethodLength
       def run_interactively
         # use tmp files to capture output and exit status of the command when
         # running interactively
         log_file = Tempfile.open('captured-output')
         exit_file = Tempfile.open('captured-exit-code')
-        Kernel.system("script -qc '#{full_command}; echo $? > #{exit_file.path}' #{log_file.path}")
+        Kernel.system(
+          "script -qc '#{full_command}; echo $? > #{exit_file.path}' | tee -i #{log_file.path}"
```

Comment:
@jjeffers I think we don't need this pull request at all? because its just extra tee command which I think is not required, because original `script` command redirects the logs in file?

Reply:
If we truly don't need it, then let's close this.

Reply:
Peter Vreeman made an observation about the need for "bash" vs "script":
```
The problem is that 'script' does not work nice togeher with nohup (or any
other standard shell backgrounding from shell) because 'script' changes the
'setsid' and detaches from the parent process when going in the background,
then the 'nohup' does not have a sub process to follow anymore
```